XCUITests/BookmarkingTests.swift

```diff
@@ -97,6 +97,41 @@ class BookmarkingTests: BaseTestCase {
         navigator.goto(MobileBookmarks)
         checkItemInBookmarkList()
     }
+    
+    func testRecentBookmarks() {
+        // Verify that there are only 4 cells without recent bookmarks
+        navigator.goto(LibraryPanel_Bookmarks)
+        navigator.nowAt(LibraryPanel_Bookmarks)
```

Comment:
I think the test will work without this, or if it does not, maybe adding a waitForExistence will make the trick. But as a first option, I will try to remove this line